.github/scripts/overwrites/stateful.sh

```diff
@@ -11,7 +11,7 @@ npm pkg set contributes.configuration[0].properties[runme.app.enableShare].defau
 npm pkg set contributes.configuration[0].properties[runme.experiments.escalationButton].default=true --json
 npm pkg set contributes.configuration[0].properties[runme.server.lifecycleIdentity].default=1 --json
 npm pkg set contributes.configuration[0].properties[runme.app.notebookAutoSave].default="yes"
-npm pkg set contributes.configuration[0].properties[runme.app.sessionOutputs].default=false --json
+npm pkg set contributes.configuration[0].properties[runme.app.sessionOutputs].default=true --json
```

Comment:
if we just want default, which is `true,` perhaps we just remove the whole line altogether?

Reply:
Fixed in 2c69b84a17dcf327bd3ee5e4e305200fb220cece